### DISCUSSION TITLE
[Issues] Petite Vue Legit bugs NOT feature requests

### DISCUSSION THREAD

**Discussion by jikkuatwork** [vuejs/petite-vue#240] [Ideas]
Since issues is disabled, lets start a (curated) discussion thread to list the issues so that @yyx990803 may get to see them easily. Lets avoid issues that will increase the scope since he feels the scope is fixed.

So, lets only post legit or functional issues.

**Comment by sqllyw** [comment]
so far I have not yet encountered a issue

**Reply by jikkuatwork** [reply]
Same here. Thats something striking for me! I have been expecting some thing or the other, but everytime its just me making a stupid mistake. Now I feel a lot of confidence using PV.

Having said that I do feel, an issues tab is important.
